origocli/data/boilerplate/pipeline/pipeline-input.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pipelineInstanceId": "PIPELINEINSTANCE",
+    "datasetUri": "input/ID/VERSION",
```

Comment:
Use `DATASET_ID` instead of just `ID`?

Reply:
Done, also updated to DATASET_VERSION to make it more explicit